internal/provision/provisionwatchers.go

```diff
@@ -96,3 +118,42 @@ func LoadProvisionWatchers(path string, dic *di.Container) errors.EdgeX {
 	_, edgexErr := pwc.Add(ctx, addProvisionWatchersReq)
 	return edgexErr
 }
+
+func processFile(lc logger.LoggingClient, path string, secretProvider interfaces.SecretProvider, addProvisionWatchersReq []requests.AddProvisionWatcherRequest) {
+	var watcher dtos.ProvisionWatcher
+	data, err := file.Load(path, timeout, secretProvider)
+	if err != nil {
+		lc.Errorf("Failed to read %s: %v", path, err)
```

Comment:
This will leak creds if path is a URI
```suggestion
		lc.Errorf("Failed to read Provision Watcher: %v", err)
```